packages/client/src/LocClient.ts

```diff
@@ -960,9 +960,9 @@ export class AuthenticatedLocClient extends LocClient {
             let verifiedThirdParty = false;
             if(request.locType === "Identity" && request.status === "CLOSED") {
                 if(availableVerifiedIssuers) {
-                    verifiedThirdParty = availableVerifiedIssuers[request.ownerAddress].find(issuer => issuer.address === request.requesterAddress) !== undefined;
+                    verifiedThirdParty = availableVerifiedIssuers[request.ownerAddress].find(issuer => issuer.address === request.requesterAddress?.address && request.requesterAddress?.type === "Polkadot") !== undefined;
                 } else {
-                    const maybeIssuer = await this.nodeApi.query.logionLoc.verifiedIssuersMap(request.ownerAddress, request.requesterAddress) as Option<PalletLogionLocVerifiedIssuer>;
+                    const maybeIssuer = await this.nodeApi.query.logionLoc.verifiedIssuersMap(request.ownerAddress, request.requesterAddress?.address) as Option<PalletLogionLocVerifiedIssuer>;
```

Comment:
We must ensure that this is only called with a Polkadot requester address.

Reply:
Correct.